src/main/java/MessageConvertor.java

```diff
@@ -1,8 +1,11 @@
 public class MessageConvertor {
-    public JsonMessage parsXmlToJson(XmlMessage a){
-        return new JsonMessage(a.getBody());
+    public JsonMessage parsXmlToJson(MessageType message) {
```

Comment:
If your method parses **xml** to Json, why does it receive any kind of MessageType?